python/scannerpy/collection.py

```diff
@@ -28,4 +27,5 @@ def profiler(self):
         return self._db.profiler(self._descriptor.job_id)
 
     def as_op(self):
-        return SamplerOp(self)
+        #return SamplerOp(self)
```

Comment:
Are collections still functional or broken atm?

Reply:
They are functional, they just aren't used in specifying jobs anymore.